src/form/Form.js

```diff
@@ -196,6 +215,49 @@
         this.btntd
             .attr("colspan", this._maxCols * 2)
         ;
+        
+        // Disable Submit unless there is data
+        if (!this.allowEmptyRequest()) {
+            this.inputs().forEach(function(input, idx) {
```

Comment:
Currently this loop get executed every update - why not just add the event handler on the enter().each() above?